frontend/src/pages/upload_photo/UploadPhoto.jsx

```diff
@@ -1,71 +1,95 @@
-import { useState } from "react";
-import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { useState, useEffect } from "react";
+import { NavContentWrapper } from "../../components/NavContentWrapper/NavContentWrapper";
+import ContentHeader from "../../components/ContentHeader/ContentHeader";
+import "./UploadPhoto.css";
 import { useNavigate } from "react-router-dom";
-import UploadArea from "../../components/UploadPhoto/UploadArea";
 import PreviewSection from "../../components/UploadPhoto/PreviewSection";
-import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
-import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos";
-import "./UploadPhoto.css";
+import UploadArea from "../../components/UploadPhoto/UploadArea";
+import { ChevronLeftIcon, ChevronRightIcon } from "@heroicons/react/24/solid";
 
-export default function UploadPhotos() {
-  const [imageFiles, setImageFiles] = useState([]);
-  const [isEditMode, setIsEditMode] = useState(false);
+export default function UploadPhoto() {
   const navigate = useNavigate();
 
-  const handleFileUpload = (file) => {
-    if (file && file.type.startsWith("image/")) {
-      const reader = new FileReader();
-      reader.onload = function (e) {
-        setImageFiles([
-          ...imageFiles,
-          { name: file.name, url: e.target.result },
-        ]);
-      };
-      reader.readAsDataURL(file);
-    } else {
-      alert("Please upload a valid image file.");
-    }
+  const handleTranscribe = () => {
```

Comment:
Can you please pull changes from the latest main onto your branch, it should have the transcriptions backend endpoint implemented. Then, please modify this function to call this endpoint, and then navigate to the '/load-transcription' route after the api call returns.

You will also want to implement the page for this route. Just a basic page with the information returned from the api call will suffice.